### ISSUE TITLE
Syntax error?

### ISSUE THREAD

**Issue by ersan** [aortbals/southwest-checkin#48]
Getting this with the latest version:

```
SyntaxError: /home/ersan/southwest-checkin/lib/southwest/reservation.rb:12: syntax error, unexpected ':', expecting =>
        'first-name': first_name,
                     ^
/home/ersan/southwest-checkin/lib/southwest/reservation.rb:13: syntax error, unexpected ':', expecting :: or '[' or '.'
        'last-name': last_name
                    ^
/home/ersan/southwest-checkin/lib/southwest.rb:5:in `require_relative'
/home/ersan/southwest-checkin/lib/southwest.rb:5:in `<top (required)>'
/home/ersan/southwest-checkin/vendor/bundle/ruby/2.1.0/gems/skylight-0.10.2/lib/skylight/probes.rb:81:in `require'
/home/ersan/southwest-checkin/vendor/bundle/ruby/2.1.0/gems/skylight-0.10.2/lib/skylight/probes.rb:81:in `require'
/home/ersan/southwest-checkin/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `block in require'
/home/ersan/southwest-checkin/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
/home/ersan/southwest-checkin/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/home/ersan/southwest-checkin/config/application.rb:36:in `<class:Application>'
/home/ersan/southwest-checkin/config/application.rb:19:in `<module:SouthwestCheckin>'
/home/ersan/southwest-checkin/config/application.rb:18:in `<top (required)>'
/home/ersan/southwest-checkin/Rakefile:4:in `require'
/home/ersan/southwest-checkin/Rakefile:4:in `<top (required)>'
```

**Comment by ersan** [comment]
Changed it to:
```
        'first-name' => first_name,
        'last-name' => last_name
```

and it works, but I'm not a ruby developer so I have no idea if that's right.

**Comment by DanG100** [comment]
Now this is the issue I've been getting

```
$ rake db:create db:migrate db:seed --trace
rake aborted!
SyntaxError: /home/ec2-user/southwest-checkin/lib/southwest/request.rb:9: syntax error, unexpected ','
    def initialize(last_name:, first_name:, record_locator:)
                              ^
/home/ec2-user/southwest-checkin/lib/southwest/request.rb:48: syntax error, unexpected keyword_end, expecting end-of-input
/home/ec2-user/southwest-checkin/lib/southwest.rb:2:in `require_relative'
/home/ec2-user/southwest-checkin/lib/southwest.rb:2:in `<top (required)>'
/home/ec2-user/.gem/ruby/2.0/gems/skylight-0.10.2/lib/skylight/probes.rb:81:in `require'
/home/ec2-user/.gem/ruby/2.0/gems/skylight-0.10.2/lib/skylight/probes.rb:81:in `require'
/home/ec2-user/.gem/ruby/2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `block in require'
/home/ec2-user/.gem/ruby/2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
/home/ec2-user/.gem/ruby/2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
/home/ec2-user/southwest-checkin/config/application.rb:36:in `<class:Application>'
/home/ec2-user/southwest-checkin/config/application.rb:19:in `<module:SouthwestCheckin>'
/home/ec2-user/southwest-checkin/config/application.rb:18:in `<top (required)>'
/usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/share/ruby/vendor_ruby/2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/home/ec2-user/southwest-checkin/Rakefile:4:in `<top (required)>'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/rake_module.rb:28:in `load'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/rake_module.rb:28:in `load_rakefile'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:689:in `raw_load_rakefile'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:94:in `block in load_rakefile'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:93:in `load_rakefile'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:77:in `block in run'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/lib/rake/application.rb:75:in `run'
/home/ec2-user/.gem/ruby/2.0/gems/rake-10.4.2/bin/rake:33:in `<top (required)>'
/home/ec2-user/bin/rake:23:in `load'
/home/ec2-user/bin/rake:23:in `<main>'
```

**Comment by ersan** [comment]
Looks like you're using Ruby 2.0 - the readme says to use 2.2 - I had some syntax errors until I switched to 2.2 so give that a shot first.

**Comment by aortbals** [comment]
Yes, ruby `2.2` or greater is required. Please reopen if that does not fix it.

**Comment by ersan** [comment]
Ruby 2.2 does not fix the error originally posted, I was responding to the guy who replied with his own error.

**Comment by aortbals** [comment]
Ah. I see. Fixed in 42f790b04740738714235e57783ca2283540c633.
